<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Probe Search</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 249, 241);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Roboto Mono for Powerline</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: white;
padding-left: 5px;
padding-right: 5px;
margin: 5px;</string>
          </property>
          <property name="text">
           <string>No files loaded</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {background-color: rgb(48, 65, 255); 
border: none;
color: white;
padding: 15px 32px;
text-align: center;
text-decoration: none;
font-size: 14px;
border-radius: 10px;
margin: 5px;
}

#pushButton:hover
{
  border: 1px solid red;
font-size: 15px;
margin: 2px;

}
#pushButton:pressed {
	border: 1px solid red;
    background-color: rgb(224, 0, 0);     
font-size: 15px;
margin: 2px;
}</string>
          </property>
          <property name="text">
           <string>Load File(s)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">padding: 10px;
background-color: rgb(230, 249, 241);
color: black;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Commands</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">padding: 10px;
background-color: rgb(230, 249, 241);
color: black;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Files</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_lw_clear">
          <property name="styleSheet">
           <string notr="true">QPushButton {
border: 1px solid red;
background-color: white;
margin: 4px;
}
#pushButton_lw_clear:hover
{
  border: 1px solid red;
font-size: 15px;
margin: 1px;

}
#pushButton_lw_clear:pressed {
	border: 1px solid red;
    background-color: rgb(224, 0, 0);     
font-size: 15px;
margin: 1px;
}</string>
          </property>
          <property name="text">
           <string>Clear Files</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QListWidget" name="listWidget_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Source Code Pro for Powerline</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #fff;
padding-left: 5px;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="font">
           <font>
            <family>Source Code Pro for Powerline</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #fff;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Roboto Mono for Powerline</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: white;
 border: 1px solid green;
margin-left: 10px;</string>
          </property>
          <property name="text">
           <string>Select command</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">padding: 10px;
background-color: rgb(230, 249, 241);
color: black;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>--&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Roboto Mono for Powerline</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: white;
padding: 10px;
border: 1px solid green;
margin-right: 10px;</string>
          </property>
          <property name="text">
           <string>Select file</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="font">
           <font>
            <family>Source Code Pro for Powerline</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #fff;
padding-left: 10px;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButton_dldHTML">
            <property name="styleSheet">
             <string notr="true">QPushButton {
background-color: rgb(48, 65, 255); 
border: none;
color: white;
padding: 15px 35px;
text-align: center;
text-decoration: none;
font-size: 15px;
border-radius: 10px;
margin: 4px;
}

#pushButton_dldHTML:hover
{
  border: 1px solid red;
font-size: 15px;
margin: 2px;

}
#pushButton_dldHTML:pressed {
	border: 1px solid red;
    background-color: rgb(224, 0, 0);     
font-size: 15px;
margin: 2px;
}</string>
            </property>
            <property name="text">
             <string>Download As HTML</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+S</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_dldCSV">
            <property name="styleSheet">
             <string notr="true">QPushButton {
background-color: rgb(48, 65, 255); 
border: none;
color: white;
padding: 15px 35px;
text-align: center;
text-decoration: none;
font-size: 15px;
border-radius: 10px;
margin: 4px;
}

#pushButton_dldCSV:hover
{
  border: 1px solid red;
font-size: 15px;
margin: 2px;

}
#pushButton_dldCSV:pressed {
	border: 1px solid red;
    background-color: rgb(224, 0, 0);     
font-size: 15px;
margin: 2px;
}</string>
            </property>
            <property name="text">
             <string>Download As CSV</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+S</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">padding: 5px;
background-color: rgb(230, 249, 241);
color: black;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Last Finished:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_last_run">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Roboto Mono for Powerline</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: white;
margin-left: 10px;</string>
          </property>
          <property name="text">
           <string>Last Run</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_exec_3">
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {background-color: #4CAF50; 
border: none;
color: white;
padding: 15px 32px;
text-align: center;
text-decoration: none;
font-size: 16px;
border-radius: 10px;
margin: 5px;
}

#pushButton_exec_3:hover
{
  border: 1px solid red;
font-size: 17px;
margin: 2px;
}
#pushButton_exec_3:pressed {
	border: 1px solid red;
    background-color: rgb(224, 0, 0);     
font-size: 17px;
margin: 2px;
}</string>
          </property>
          <property name="text">
           <string>           Execute Command           </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Files"/>
    <addaction name="actionSave_Defaults"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Files">
   <property name="text">
    <string>Load Files</string>
   </property>
  </action>
  <action name="actionSave_Defaults">
   <property name="text">
    <string>Save Defaults</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
